File: Java/1222.java
import java.util.Scanner;

public class Main{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int time = sc.nextInt();
		int a = sc.nextInt();
		int b = sc.nextInt();
		
		for(int i = time; i < 90; i+=5){
			a++;
		}
		
		if(a > b) System.out.println("win");
		else if(a < b) System.out.println("lose");
		else System.out.println("same");
	}
}

File: C++/1222.cpp
#include <iostream>

using namespace std;

int main(){
	int time, a, b;
	cin >> time >> a >> b;
	
	for(int i = time; i < 90; i+=5){
		a++;
	}
	
	if(a > b) cout << "win";
	else if(a < b) cout << "lose";
	else cout << "same";
}